docs/docs/guides/smart_contracts/writing_contracts/notes/custom_note.md

```diff
@@ -2,7 +2,7 @@
 title: Using custom note types in Aztec.nr
 ---
 
-It may be useful to write a custom note type if you want to use a specific type of private data or struct that does not have a default implementation in Aztec.nr. Using a custom note type means that the entire note is nullified with one nullifier, which may be better for your circumstances than having multiple notes that need nullified. 
+It may be useful to write a custom note type if you want to use a specific type of private data or struct that does not have a default implementation in Aztec.nr. Using a custom note type means that the entire note can be nullified with one nullifier, which may be better for your circumstances than having multiple notes that need nullified. 
```

Comment:
"entire note" nullification still isn't clear. Suggesting analogy to "singleton" if that is what you mean? But you possibly mean one nullifier for entire note type, and not multiple unique nullifiers per note?
```suggestion
It may be useful to write a custom note type if you want to use a specific type of private data or struct that does not have a default implementation in Aztec.nr. Using a custom note type also means that note can be like a singleton and thus only one nullifier, which may be better for your circumstances than having multiple notes that need nullification.
```

Reply:
well no, not like a singleton, like a note. you could for example use 3 privateImmutables instead of creating a note with three values. but when these values are spent you would have to create three nullifiers for each of the privateImmutables. when using one note they can all be nullified